src/replit/database/server.py

```diff
@@ -4,7 +4,7 @@
 
 from flask import Blueprint, Flask, request
 
-from .default_db import db
+from . import default_db
```

Comment:
Is it even possible to use a db other than the default_db?

Reply:
Yes! As an example, we use it in some CI tests internally. An example is as follows (I added a `print` in the default DB initializer):

```
>>> import os
>>> from replit.database import Database
>>> db = Database(os.environ.get("REPLIT_DB_URL"))
>>> db.set('foo', 'bar')
>>> db.get('foo')
'bar'
>>> import replit
>>> replit.db
Initializing default db!
<Database(db_url=...)>
```

You can see in this example that the initializer for the default database is only triggered on certain property accessors.